### PR TITLE
added a random function to randomly distribute news entries

### LINKED ISSUE #346: Make news to get random entry like kiosk
https://github.com/f-klubben/stregsystemet/blob/9748ebebecc7cdb8fca69457aad70fa1a86924a2/stregsystem/views.py#L55

Multiple news might want to be shown at same time, randomly pick upon load which one to show. 

Currently, if more than one exist for some timestamp, we get an error on ORM filter `.get()` which blocks stregsystemet from loading.

### PR DESCRIPTION
**Pull request by Dev-Destructor**
fixes #346

### CODE REVIEW

**Review by VirtualSatai — APPROVED** [review]
LGTM :shipit:

File: stregsystem/views.py [hunk 1]
```diff
@@ -54,7 +55,10 @@
 
 def __get_news():
     try:
-        return News.objects.filter(stop_date__gte=timezone.now(), pub_date__lte=timezone.now()).get()
+        current_time = timezone.now()
+        news_items = News.objects.filter(stop_date__gte=current_time, pub_date__lte=current_time)
```
Comment (falkecarlsen) [inline stregsystem/views.py:59]:
```suggestion
        news_items = News.objects.filter(stop_date__gte=current_time, pub_date__lte=current_time).order_by('?').first()
```
https://docs.djangoproject.com/en/dev/ref/models/querysets/#django.db.models.query.QuerySet.order_by

**Review by falkecarlsen — CHANGES_REQUESTED** [review]
Doesn't pass tests as the empty set, as random.choice([]) is an error

**Comment by Dev-Destructor** [comment]
> Doesn't pass tests as the empty set, as random.choice([]) is an error

Yeah, Sorry

**Review by falkecarlsen — APPROVED** [review]
Removed unnecessary random.choice call on list of one elem.

Thanks :shipit:

**Pull request merged**